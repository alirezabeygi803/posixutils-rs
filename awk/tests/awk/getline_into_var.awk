{
	print $0;
	print NF, NR, FNR;
	getline next_record;
	print NF, NR, FNR;
	print $0;
	print next_record;
}

{
	print NF, NR, FNR;
	print $0;
}