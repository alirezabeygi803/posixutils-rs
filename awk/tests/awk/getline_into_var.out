1 Jane janitor 30
4 1 1
4 2 2
1 Jane janitor 30
2 Smith sailor 45
4 2 2
1 Jane janitor 30
3 Roselyn radiologist 28
4 3 3
4 4 4
3 Roselyn radiologist 28
4 Megan machinist 32
4 4 4
3 Roselyn radiologist 28
5 Celine clergy 87
4 5 5
4 6 6
5 Celine clergy 87
6 Fredrick farmer 19
4 6 6
5 Celine clergy 87
7 Benjamin programmer 53
4 7 7
4 7 7
7 Benjamin programmer 53
6 Fredrick farmer 19
4 7 7
7 Benjamin programmer 53
